workflows/argo/offline-diags.yaml

```
@@ -56,14 +71,24 @@ spec:
             cat << EOF > training_data.yaml
             {{inputs.parameters.training_data_config}}
             EOF
+
+            if [ "{{inputs.parameters.use-wandb}}" == "false" ];
+            then
+              wandb_flag=" "
```

Would be nice to settle whether these empty strings are necessary or not. If they are then can you add a comment saying why?
```suggestion
              wandb_flag=""
```

---

They aren't necessary, I think I just do " " on autopilot sometimes.